ci/do_ci.sh

```diff
@@ -397,12 +397,13 @@ elif [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
   echo "Building envoy fuzzers and executing 100 fuzz iterations..."
   bazel_with_collection test "${BAZEL_BUILD_OPTIONS[@]}" --config=asan-fuzzer "${FUZZ_TEST_TARGETS[@]}" --test_arg="-runs=10"
   exit 0
+elif [[ "$CI_TARGET" == "format_pre" ]]; then
+  BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS[*]}" ./ci/format_pre.sh
 elif [[ "$CI_TARGET" == "fix_format" ]]; then
   # proto_format.sh needs to build protobuf.
   setup_clang_toolchain
 
   echo "fix_format..."
-  ./tools/code_format/check_shellcheck_format.sh fix
```

Comment:
we lose the shellcheck autofix here but atm it doesnt actually apply any diff/fix (the fixes are unreliable and diffing is disabled)

Reply:
How is this related to the objective of this change?

Reply:
shellcheck runs fast and the output is buried in logs with existing setup

Reply:
also with current format check it runs and if *any* fails it runs again with fix code

this means that if shellcheck fails it re-runs all the other checkers even tho shellcheck wont provide any diff